File: compiler-group-traslator/src/test/resources/IntAdd.java
public class IntAdd {
    public static void main(String[] args) {
        StaticJavaLib.assertTrue( 2 +  3 ==  5);
        StaticJavaLib.assertTrue( 2 + -3 == -1);
        StaticJavaLib.assertTrue(-2 +  3 ==  1);
        StaticJavaLib.assertTrue(-2 + -3 == -5);

        StaticJavaLib.assertTrue(add( 2,  3) ==  5);
        StaticJavaLib.assertTrue(add( 2, -3) == -1);
        StaticJavaLib.assertTrue(add(-2,  3) ==  1);
        StaticJavaLib.assertTrue(add(-2, -3) == -5);
    }
    
    static int add(int i, int j) {
        return i + j;
    }
}

File: compiler-group-traslator/src/test/resources/c/IntAdd.c
#include<stdio.h>
#include<stdlib.h>
#include<stdbool.h>
#include "GC.c"
#include "ExceptionHandlingFunctions.c"
VM* vm;
int add(int i, int j) {
	return i + j;

	//set all vars to zero and have the TEMP variable for the return expression
}
int main(char* args) {
	vm = newVM();
	assertTrue(2 + 3 == 5);
	assertTrue(2 + -3 == -1);
	assertTrue(-2 + 3 == 1);
	assertTrue(-2 + -3 == -5);
	assertTrue(add(2, 3) == 5);
	assertTrue(add(2, -3) == -1);
	assertTrue(add(-2, 3) == 1);
	assertTrue(add(-2, -3) == -5);
	return 0;
}